zeppelin-interpreter/pom.xml

```diff
@@ -98,6 +100,69 @@
       <artifactId>slf4j-log4j12</artifactId>
     </dependency>
 
+    <!-- Copycat is a fault-tolerant state machine replication framework. -->
+    <dependency>
+      <groupId>io.atomix.copycat</groupId>
```

Comment:
Isn't this dependency moved to `io.atomix`?

Reply:
Sorry, I didn't understand what you mean.